projects/ngx-charts-on-fhir/src/lib/data-layer-list/annotation-list/annotation-list.component.css

```diff
@@ -1,5 +1,4 @@
 .scrolling-list {
-  max-height: 500px;
   overflow-y: auto;
 }
 
```

Comment:
the whole rule can be removed because overflow-y has no effect without max-height.

Reply:
and please remove the `class="scrolling-list"` from html

Reply:
ok colin